sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -120,6 +119,9 @@ def test_sanity_check_pls_regression():
     assert_array_almost_equal(x_loadings_sign_flip, x_weights_sign_flip)
     assert_array_almost_equal(y_loadings_sign_flip, y_weights_sign_flip)
 
+    assert_allclose(X_trans, pls.x_scores_)
+    assert_allclose(Y_trans, pls.y_scores_)
```

Comment:
@NicolasHug It seems that this will fail while the documentation is stating the equivalence. Since I am not familiar with this method, could you bring some light here?

Reply:
Could this be a doc issue that's the same as https://github.com/scikit-learn/scikit-learn/pull/21585#issuecomment-962641396 ?

Reply:
The docstring (not online doc) is stating:

```python
y_scores_ : ndarray of shape (n_samples, n_components)
    The transformed training targets.
```

Going in the code, I see that there is potentially a normalization factor that exist for `y_scores_` but not `x_scores_` but I don't know anything about the algorithm :)